src/common/transformations/src/transformations/common_optimizations/common_optimizations.cpp

```diff
@@ -164,6 +165,7 @@ bool ov::pass::CommonOptimizations::run_on_model(const std::shared_ptr<ov::Model
     ADD_MATCHER(decomp, ConvertDivide)
     ADD_MATCHER(decomp, ConvertDepthToSpace)
     ADD_MATCHER(decomp, ConvertSpaceToDepth)
+    ADD_MATCHER(decomp, ConvertConvertPromoteTypes)
```

Comment:
As the transformation is going to replace frontend mechanism, maybe it should be a part of the MOCTransfromations as well.
To be verified.

Reply:
My idea was to replace existing internal transformation in PT FE in separate ticket. Not sure whether it would be also required in MO, but I can verify it and add in separate ticket if needed